version

```diff
@@ -0,0 +1 @@
+2.0.0
```

Comment:
What is the purpose of this file?

Reply:
Just externalized the version. The gemspec now refers to this file.